support/azure/azure-kubernetes/availability-performance/container-image-pull-performance.md

```diff
@@ -48,8 +50,32 @@ Switch to newer hardware generation virtual machine (VM) stock keeping units (SK
 
 Increase the available CPU cores on your VM SKU to reduce a CPU bottleneck in image pulling. Allocating more cores helps avoid throttling on the VM.
 
-### Switch back to serialized image pulls
+### Toggle image pull type
+
+Use the following AFEC flag instructions to toggle parallel image on or off. 
+
+When you run az feature list --namespace Microsoft.ContainerService, you should see the following feature appear:
```

Comment:
When you run the `az feature list --namespace Microsoft.ContainerService` command, you...